cmd/oras/internal/display/status/console/console_test.go

```diff
@@ -96,7 +96,7 @@ func TestConsole_NewRow(t *testing.T) {
 
 	c.NewRow()
 
-	err := testutils.MatchPty(pty, tty, "^[8\r\n^[7")
+	err := testutils.MatchPty(pty, tty, "\x1b8\r\n\x1b7")
```

Comment:
Just a non-blocking question, how does the fix work? 🧐

Reply:
The old code is ascii control character and the new code is the escape sequence for that characters which is better IMO..